storm-core/src/jvm/org/apache/storm/trident/Stream.java

```diff
@@ -124,6 +124,31 @@ public Stream parallelismHint(int hint) {
     }
 
     /**
+     * Sets the CPU Load resource for the current operation
+     */
+    public Stream setCPULoad(Number load) {
```

Comment:
Can we make Stream implement `ResourceDeclarer<Stream>` and have these marked with @Override.